structure.md

```diff
@@ -0,0 +1,111 @@
+# Structure
```

Comment:
This probably wasn't mean to be included in the commit.

We will need to write something similar with the table for a v1 to v2 migration guide, though.